routing/route/route.go

```diff
@@ -108,9 +108,18 @@ func (h *Hop) PackHopPayload(w io.Writer, nextChanID uint64) error {
 	combinedRecords := append(h.TLVRecords,
```

Comment:
Is something bad happening here because the underlying slice array is modified?

Reply:
fixed